index.html

```diff
@@ -67,6 +67,10 @@ <h2>Registration Entry Requirements and Update Process</h2>
 			<li>Each entry should include a link that references a public available
 			specification (PAS) that defines the associated interface type.</li>
 			<li>Each entry must include contact information of the requestor.</li>
+                        <li>Each entry must include an <dfn data-cite=
+		'PERFORMANCE-TIMELINE-2#dom-performancetimeline-availableFromTimeline'><code>availableFromTimeline</code></dfn> boolean.
```

Comment:
The link is broken. Also, I think we have this inverted? That is, the PerformanceTimeline value should reference this one, not the other way around.

Reply:
Right, this PR is actually gated on the PerformanceTimeline pull since there isn't currently an anchor to these dfn's: https://github.com/w3c/performance-timeline/pull/135

I know we discussed the PerformanceTimeline spec referencing this one, but after looking back at the changes you recently made to PerformanceTimeline as well as what I have there with these two, I think it makes more sense this way.  We already reference entryType from that CL and I think we would have to move a lot of context into this spec about observers and buffered flags to have it make sense.

Reply:
I think adding some of that context actually makes sense here. The PerformanceTimeline value must reference this one, since this is the one which contains the values for each entryType. So it seems circular if this one also references the PerformanceTimeline one? @yoavweiss thoughts?

Reply:
I think it'd be better to define a variable with the same name here and link to it from performanceTimeline, rather than the other way around

Reply:
Done.